devito/core/operator.py

```diff
@@ -1,19 +1,20 @@
 from devito.core.autotuning import autotune
-from devito.dle import NThreads
 from devito.ir.support import align_accesses
 from devito.parameters import configuration
 from devito.operator import Operator
+from devito.targets import NThreads
 
 __all__ = ['OperatorCore']
 
 
 class OperatorCore(Operator):
 
-    def _specialize_exprs(self, expressions):
+    @classmethod
```

Comment:
a very long story.... but basically I need `__new__` instead of `__init__`. So everything called from the former `__init__` needs to be turned into a `classmethod`. Note this is also much more conceptually correct than before: now we are incrementally building a Callable/Operator, while previously we were initializing its state (body). There is a subtle yet meaningful difference